src/installer/pkg/sfx/installers/host.wxs

```diff
@@ -79,34 +89,43 @@
 
   <Fragment>
     <Property Id="DISABLE_SETTING_HOST_PATH" Secure="yes">
-      <RegistrySearch Id="DisableSettingHostPathSearch" Root="HKLM" Key="SOFTWARE\Microsoft\.NET" Type="raw" Name="DisableSettingHostPath"/>
+      <RegistrySearch Id="DisableSettingHostPathSearch" Root="HKLM" Key="SOFTWARE\Microsoft\.NET" Type="raw" Name="DisableSettingHostPath" />
     </Property>
 
-    <Component Id="cmpSetPath" Guid="{0B910ED8-0877-473D-8658-647382324433}" Directory="DOTNETHOME">
-      <CreateFolder />
-      <!-- Always set the SYSTEM PATH, unless DisableSettingHostPath is 1. -->
+    <?if $(var.Platform)~=x64 ?>
```

Comment:
Is `Platform` consistent with `InstallerPlatform` (that's the new variable WiX will use). You can use something else, but the WiX SDK doesn't set that AFAIK. Just double checking.

Reply:
`-arch` to wix.exe seems to cause this variable to be populated.